collection/sites/googledocs.yml

```diff
@@ -0,0 +1,11 @@
+name: All Google Documents (disable Google Sheets before enable it!)
```

Comment:
> disable Google Sheets before enable it!

I'm not sure about this as end-user will never see the comment. Will this cause some conflict or something?

Reply:
The user will see the comment in the list of sites.
So I realized that the conflict is due to the same host, docs.google.com. Without this host support for documents/presentations/forms cannot be added

Reply:
Okay, tried and it doesn't seem to break the board.

Reply:
Simultaneous enabling functions does not cause errors. Just links to documents, presentations and forms do not work